polaris-react/src/components/Page/components/Header/Header.scss

```diff
@@ -15,6 +15,29 @@ $action-menu-rollup-computed-width: 40px;
 
 .BreadcrumbWrapper {
   grid-area: breadcrumbs;
+
+  #{$se23} & {
+    /* stylelint-disable-next-line selector-max-combinators -- SE23 match button group */
+    a,
```

Comment:
The [other buttons in the page header are styled with overrides](https://github.com/Shopify/polaris/blob/4d018664742bc3c9e7c4b577d31dcd05d09a2a6f/polaris-react/src/components/ActionMenu/components/SecondaryAction/SecondaryAction.scss#L8) so I matched that pattern but open to another suggestion as well